contracts/strategies/uniswap-gamma/UniswapGammaStrategy.sol

```diff
@@ -0,0 +1,313 @@
+//SPDX-License-Identifier: Unlicense
+
+pragma solidity 0.6.12;
+
+import "@openzeppelin/contracts/math/Math.sol";
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+import '@uniswap/v3-core/contracts/interfaces/IUniswapV3Pool.sol';
```

Comment:
Can you copy the contract into `base/interface/uniswap` and remove the dependency?